<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyThings</class>
 <widget class="QDialog" name="MyThings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="recordsList">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>711</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 238, 196);
color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(205, 201, 255);
font: 11pt &quot;清松手寫體2&quot;;</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="CustomLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-9</y>
     <width>921</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 251, 183, 60%);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="leave">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>380</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>離開</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="_src.qrc">
     <normaloff>:/new/prefix1/gp_source/leave.png</normaloff>:/new/prefix1/gp_source/leave.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>62</width>
     <height>62</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="CustomerLineEdit" name="searchKey">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="search_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>382</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 10pt &quot;timemachine wa&quot;;</string>
   </property>
   <property name="text">
    <string>搜尋模式:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;搜尋事項&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="_src.qrc">
     <normaloff>:/new/prefix1/gp_source/search.png</normaloff>:/new/prefix1/gp_source/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="proBar">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>436</y>
     <width>53</width>
     <height>6</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>380</y>
     <width>1</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 152, 152);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="modeChoose">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <item>
    <property name="text">
     <string>事項</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小於優先序</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>大於優先序</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="alert_tip">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>435</y>
     <width>288</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(172, 37, 39);</string>
   </property>
   <property name="text">
    <string>請輸入正整數</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>557</x>
     <y>380</y>
     <width>1</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 152, 152);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="myAlerm">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>380</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>我的鈴聲</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="_src.qrc">
     <normaloff>:/new/prefix1/gp_source/bell.png</normaloff>:/new/prefix1/gp_source/bell.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>background</zorder>
  <zorder>recordsList</zorder>
  <zorder>leave</zorder>
  <zorder>searchKey</zorder>
  <zorder>search_label</zorder>
  <zorder>search</zorder>
  <zorder>proBar</zorder>
  <zorder>modeChoose</zorder>
  <zorder>alert_tip</zorder>
  <zorder>myAlerm</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomerLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customerlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="_src.qrc"/>
 </resources>
 <connections/>
</ui>
